src/components/Settings/Settings.js

```diff
@@ -40,78 +50,103 @@ const Settings = ({
     onSettingsChange({ gitHubAuthorized: false });
   };
 
-  const repoChangeHandler = _.compose(
-    _.debounce(onRepoQueryChange, 500),
-    _.property('value'),
-    _.property('target')
-  );
+  const repoChangeHandler =
+    _.debounce(onRepoQueryChange, 500)
+  ;
 
   return (
-    <div style={styles.board}>
+    <div className='settings-container'>
       <HeaderBar
         heading='Swimlanes Settings'
         sidebarVisible={false}
         showFilter={false}
         showSettings={false} />
-      <h1>Swimlanes Settings</h1>
       <form>
+        <h3 style={{color:'#CCCCCC',borderBottom:'1px solid #CCCCCC'}}>Pivotal settings</h3>
```

Comment:
let's add a shared style in the `styles` object above so we can reuse this header style for the other header